src/Sentry.Profiling/ProfilingIntegration.cs

```diff
@@ -7,12 +7,32 @@ namespace Sentry.Profiling;
 /// </summary>
 public class ProfilingIntegration : ISdkIntegration
 {
+    private TimeSpan _startupTimeout;
+
+    /// <summary>
+    /// Initializes the profiling integration.
+    /// </summary>
+    /// <param name="startupTimeout">
+    /// If not given or TimeSpan.Zero, then the profiler initialization is asynchronous.
+    /// This is useful for applications that need to start quickly. The profiler will start in the background
+    /// and will be ready to capture transactions that have started after the profiler has started.
+    ///
+    /// If given a non-zero timeout, profiling startup blocks up to the given amount of time. If the timeout is reached
+    /// and the profiler session hasn't started yet, the execution is unblocked and behaves as the async startup,
+    /// i.e. transactions will be profiled only after the session is eventually started.
+    /// </param>
+    public ProfilingIntegration(TimeSpan startupTimeout = default)
+    {
+        Debug.Assert(TimeSpan.Zero == default);
```

Comment:
Curious to know what the purpose of this line is... is that in case Microsoft ever change the default value for `TimeSpan` (which would implicitly break our functionality)?

Reply:
It communicates the assumptions made later in the code. For me, it wasn't 100 % clear what a default value for `TimeSpan` is so this clarifies it for anyone else too (and for me later when I forget :) )